datacapture/src/test/java/com/google/android/fhir/datacapture/QuestionnaireItemAdapterTest.kt

```diff
@@ -234,6 +234,58 @@ class QuestionnaireItemAdapterTest {
         )
     }
 
+    @Test
+    fun getItemViewType_choiceItemType_shouldReturnRadioGroupViewHolderTypeWithExtension() {
+        val questionnaireItemAdapter = QuestionnaireItemAdapter()
+        questionnaireItemAdapter.submitList(
+            listOf(
+                QuestionnaireItemViewItem(
+                    Questionnaire.Item.newBuilder()
+                        .setType(
+                            Questionnaire.Item.TypeCode.newBuilder()
+                                .setValue(QuestionnaireItemTypeCode.Value.CHOICE)
+                        )
+                        .addExtension(
+                            Extension.newBuilder()
+                                .setUrl(
+                                    Uri.newBuilder()
+                                        .setValue(EXTENSION_ITEM_CONTROL_URL)
+                                )
+                                .setValue(
+                                    Extension.ValueX.newBuilder()
+                                        .setCodeableConcept(
+                                            CodeableConcept.newBuilder()
+                                                .addCoding(
+                                                    Coding.newBuilder()
+                                                        .setCode(
+                                                            Code.newBuilder()
+                                                                .setValue(ITEM_CONTROL_RADIO_BUTTON)
+                                                        )
+                                                        .setDisplay(
+                                                            String.newBuilder()
+                                                                .setValue("Radio Button")
+                                                        )
+                                                        .setSystem(
+                                                            Uri.newBuilder()
+                                                                .setValue(
+                                                                    EXTENSION_ITEM_CONTROL_SYSTEM
+                                                                )
+                                                        )
+                                                )
+                                        )
+                                )
+                        )
+                        .build(),
+                    QuestionnaireResponse.Item.newBuilder()
+                ) {}
+            )
+        )
+
+        assertThat(questionnaireItemAdapter.getItemViewType(0))
+            .isEqualTo(QuestionnaireItemViewHolderType.RADIO_GROUP.value
+            )
+    }
+
     @Test
     fun getItemViewType_choiceItemType_shouldReturnDropDownViewHolderTypeWithoutExtension() {
```

Comment:
similarly please update the test name:

```suggestion
    fun getItemViewType_choiceItemType_moreAnswerOptions_shouldReturnDropDownViewHolderType() {
```

Reply:
and for the next test case

fun getItemViewType_choiceItemType_itemControlExtensionWithDropDown_shouldReturnDropDownViewHolderType() {